assets/queries/terraform/aws/ec2_not_ebs_optimized/query.rego

```diff
@@ -61,3 +67,11 @@ CxPolicy[result] {
 		"searchLine": common_lib.build_search_line(["module", name], []),
 	}
 }
+
+# Since terraform does not provide a default value for instance_type, we use the default value defined on cloud formation
+get_instance_type(instanceProperties, instanceKey) = result {
+	common_lib.valid_key(instanceProperties, instanceKey)
+	result = instanceProperties[instanceKey]
+} else {
+	result = "m1.small"
+}
```

Comment:
First of all, thanks for the kind message and I hope you all are doing well. 🙂❤️

About this code section, I understand `instance_type` will be filled when doing `terraform apply`, but I think, in this cause, `launch_template` should be checked in case of `instance_type` is missing, but this check can transform this query in a very complex one, this is why I tried to do in a simpler way.

Besides that, if there is no problem in not checking with a default value, I don't have a problem to remove this check either. 🙂

Reply:
I am. Thank you so much, @felipe-avelar! ❤️

We think there is no problem in not checking with a default value 🙂

Reply:
Done 🙂